server/test/instant/db/instaql_test.clj

```diff
@@ -4253,6 +4253,9 @@
                                  :bookshelves {:allow {:view "data.name == 'Nonfiction'"}}
                                  :books {:allow {:view "data.isbn13 == '9780316486668'"}}}})
 
+        ;; prevent attrs lookup from messing with query count
+        (attr-model/get-by-app-id (:id app))
```

Comment:
It might work a little better to put the `(make-ctx)` in a `let` that wraps the `(binding [sql/*in-progress-stmts* query-tracker]` instead.